marimbabot_audio/launch/marimbabot.launch

```diff
@@ -1,14 +1,19 @@
 <?xml version="1.0"?>
 <launch>
-    <node name="audio_from_lilypond" pkg="marimbabot_audio" type="audio_from_lilypond.py"/>
-    <node name="audio_audio_capture" pkg="audio_capture" type="audio_capture" output="screen">
-        <param name="device" value="" />
-        <param name="format" value="wave" />
-        <param name="channels" value="1" />
-        <param name="depth" value="16" />
-        <param name="sample_rate" value="44100" />
-    </node>
+
+  <!-- list the devices: arecord -l -->
+    <group ns="audio_node">
+      <node name="note_audio_capture" pkg="audio_capture" type="audio_capture" output="screen">
+          <param name="device" value="plughw:3,0" />
+          <param name="format" value="wave" />
+          <param name="channels" value="1" />
+          <param name="depth" value="16" />
+          <param name="sample_rate" value="44100" />
+      </node>
+    </group>
     <node name="onset_detector" pkg="marimbabot_audio" type="onset_detection.py" output="screen" />
     <node name="onset_midi" pkg="marimbabot_audio" type="note2midi.py" output="screen" />
     <node name="soundplay_node" pkg="sound_play" type="soundplay_node.py"/>
+    <node name="audio_from_lilypond" pkg="marimbabot_audio" type="audio_from_lilypond.py"/>
+    <node name="audio_node_rviz" pkg="rviz" type="rviz" args="-d $(find marimbabot_audio)/config/config.rviz"/>
```

Comment:
Do we want to launch rviz every time?

Reply:
ok, let me create a new launch file for exhibition, like the vision node.